src/commands/clone.js

```diff
@@ -102,13 +112,19 @@ export async function _clone({
       remote,
       noCheckout,
     })
+    return { basename: dirname }
```

Comment:
I would make the API always return basename, but not name of the directory only a full path. But first of all you can change `.git` when using bare repo, so this will break existing API. You need to move `createDir` into `git.clone()` not library's internal function.